### PR TITLE
8927 - Fix for Org parties

### PR DESCRIPTION
**Pull request by andrepestana-aot**
Issue #: /bcgov/entity#8927

Description of changes:

- Changed partyType from `org` to `organization`;
- Changed property from `orgName` to `organizationName`;
- Added tests
- Updated schema version

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

**Comment by andrepestana-aot** [comment]
I think it's done when we can merge it to main. I'm keeping it as DRAFT to avoid merging now.

I don't know why `tests/unit/resources/v2/test_business_filings/test_filings.py` is failing on CI env. It doesn't fail on local tests...

File: legal-api/tests/unit/resources/v1/test_business_filings/test_filings.py [hunk 1]
```diff
@@ -110,6 +113,68 @@ def test_get_one_business_filing_by_id(session, client, jwt):
     assert rv.json['filing']['business'] == ANNUAL_REPORT['filing']['business']
 
 
+def test_business_filing_ia_parties(session, client, jwt):
+    """Assert that the business info cannot be received in a valid JSONSchema format."""
```
Comment (severinbeauvais) [inline legal-api/tests/unit/resources/v1/test_business_filings/test_filings.py:117]: "Can" be received?

File: queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/__init__.py [hunk 2]
```diff
@@ -92,15 +92,16 @@ def create_party(business_id: int, party_info: dict, create: bool = True) -> Par
...
             middle_initial=party_info['officer'].get('middleInitial', '').upper(),
-            org_name=party_info.get('orgName', '').upper()
+            org_name=party_info['officer'].get('organizationName', '').upper()
         )
     if not party:
         party = Party(
             first_name=party_info['officer'].get('firstName', '').upper(),
             last_name=party_info['officer'].get('lastName', '').upper(),
             middle_initial=party_info['officer'].get('middleInitial', '').upper(),
             title=party_info.get('title', '').upper(),
```
Comment (severinbeauvais) [inline queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/__init__.py:102]: Title? Is this used anywhere?

**Review by severinbeauvais — APPROVED** [review]
LGTM. It's nice to see this new consistency :)

I think you'll probably have to fix the tests before merging.

Also, you said that you don't think this will break the FE if deployed first (to Dev initially)?

File: queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/__init__.py (hunk 2 above)
Comment (andrepestana-aot) [reply queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/__init__.py:102]: I'm not sure but it is still in the schema: https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/party.json#L98

File: legal-api/tests/unit/resources/v1/test_business_filings/test_filings.py (hunk 1 above)
Comment (andrepestana-aot) [reply legal-api/tests/unit/resources/v1/test_business_filings/test_filings.py:117]: Yes... Will change the comments. Thanks.

**Comment by andrepestana-aot** [comment]
> Also, you said that you don't think this will break the FE if deployed first (to Dev initially)?

It will break the FE. I tested locally.

**Comment by andrepestana-aot** [comment]
Changes for the UIs are done! Just waiting for this one to be merged:
- https://github.com/bcgov/business-create-ui/pull/248
- https://github.com/bcgov/business-edit-ui/pull/209
- No changes needed for business-filings-ui

**Comment by severinbeauvais** [comment]
Are the linting and tests broken due to this code change? If so, can you please fix that? (Check with Thor for the final word on this.)

**Review by thorwolpert — APPROVED** [review]

**Review by thorwolpert — COMMENTED** [review]
lgtm, just need to handle the CI errors

**Pull request merged**